db/migrate/20220806200248_set_is_tracker_as_banned_to_domains.rb

```diff
@@ -0,0 +1,10 @@
+class SetIsTrackerAsBannedToDomains < ActiveRecord::Migration[7.0]
+  def up
+    if ActiveRecord::Base.connection.column_exists?(:domains, :is_tracker)
```

Comment:
Please remove this `if`. If the migration is run against a db that doesn't have this column, I want it to throw errors and fail.

Reply:
OK